### PR TITLE
Packaging and CI

### PR DESCRIPTION
**Pull request by jsturdy**
## Description
PR includes changes to build RPM packages, as well as `pip` installable packages.

### Types of changes
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context
While not strictly a breaking change (no functionality has been broken) it is incompatible with previous code developed in this codebase (and is likewise currently incompatible with the `develop` branch).

## How Has This Been Tested?
* TravisCI testing of the non-executed parts of this code have been done
* Further testing needs to be done in a production environment where a complete workflow can be tested
Related to cms-gem-daq-project/gem-plotting-tools#89

### CODE REVIEW

**Comment by jsturdy** [comment]
Conflicts have been resolved on the command line and are awaiting a review for a push

**Review by mexanick — COMMENTED** [review]
Basically, the comments are the same as in gemplotting...

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,10 @@
 # Don't ignore special files
 !*.md
 !*.rst
+!LICENSE
+
+# Don't ignore makefiles
+!Makefile
```
Comment (bdorney) [inline .gitignore]: related [comment](https://github.com/cms-gem-daq-project/gem-plotting-tools/pull/89/files#r183970178) on `gem-plotting-tools` PR

File: README.md [hunk 2]
```diff
@@ -1,4 +1,12 @@
 # vfatqc-python-scripts
+[![Coverage Status](https://coveralls.io/repos/github/jsturdy/vfatqc-python-scripts/badge.svg?branch=master)](https://coveralls.io/github/jsturdy/vfatqc-python-scripts?branch=master)
+[![Coverage Status](https://coveralls.io/repos/github/jsturdy/vfatqc-python-scripts/badge.svg?branch=develop)](https://coveralls.io/github/jsturdy/vfatqc-python-scripts?branch=develop)
+
+Branch|[![Travis CI logo](TravisCI.png)](https://travis-ci.org)|[![Coveralls logo](Coveralls.png)](https://www.coveralls.io)|[![Codecov logo](Codecov.png)](https://www.codecov.io)|[![Codacy logo](Codacy.png)](https://www.codacy.com)|[![Landscape logo](Landscape.png)](https://www.landscape.io)|[![CodeClimate logo](Codeclimate.png)](https://www.codeclimate.com)
```
Comment (bdorney) [inline README.md]: same comment as [this one](https://github.com/cms-gem-daq-project/gem-plotting-tools/pull/89/files#r183973319)

**Review by bdorney — COMMENTED** [review]
same comment as on `gem-plotting-tools`.  Best to continue the discussion there.

**Comment by jsturdy** [comment]
Will do a command line merge of this, also updating at the same time the `jsturdy` specifics

**Pull request merged**